tools/crd-bumper/pkg/vendoring.py

```diff
@@ -83,6 +83,8 @@ def set_preferred_api_alias(self, alt_main_file=None):
         """
 
         def search_for_alias(file_name):
+            if not os.path.exists(file_name):
+                return
```

Comment:
do we need an error message here?

Reply:
Nope. That "cmd/main.go" file doesn't exist in the nnf-integration-test repo.